src/views/mainLayout.js

```diff
@@ -35,14 +35,27 @@ const mainLayout = data => `
 
     <script src='/js/index.js' type='module'></script>
 
-    <!-- Google Tag Manager -->
-    <script nonce='${data.nonce}'>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-    'https://www.googletagmanager.com/gtm.js?id='+i+dl;var n=d.querySelector('[nonce]');
-    n&&j.setAttribute('nonce',n.nonce||n.getAttribute('nonce'));f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','${AppConstants.GA4_MEASUREMENT_ID}');</script>
-    <!-- End Google Tag Manager -->
+    <!-- Google tag (gtag.js) -->
+    <script nonce='${data.nonce}' type='module'>
+      import { dntEnabled } from '/js/analytics_dnt-helper.js'
```

Comment:
This import works in dev because the modules are not bundled/minified/etc.  However, this doesn't work for production (`npm start`).  To make a module available for import in prod with current setup, we need to decide if it's specific to a partial, or site-wide.  
- If specific to a partial, we could add to `/js/partials/`, ESBuild will process/minify it, and you include the script tag in the partial HTML.
- If site-wide (which I would say this is), we would simply import like the other scripts it in `/js/index.js`.

HOWEVER, the way you're using this in a separate script means that importing it into index.js is not going to be a slam dunk.  You could actually do this in index.js:
`export { dntEnabled } from './analytics_dnt-helper.js'`
Then, you could import it in your separate script tag like so:  `import { dntEnabled } from '/js/index.js'`

But as I investigated all this looking for an easier, more current way of accomplishing our goal, I think it would be more performant and less complex to simply test for `navigator.doNotTrack`, and remove the entire `analytics_dnt-helper.js` module.  I suggest removing this module for a couple reasons:
- it's old and possibly outdated
- we don't support internet explorer
- it's mostly based on `navigator.doNotTrack` anyway, but tries to make it friendly to other browsers, however `navigator.doNotTrack` [has been deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack), so really this module seems much ado about nothing.

Sorry for the word-wall.  I pushed up my suggestion here: https://github.com/mozilla/blurts-server/pull/2916/commits/89012aff6ac237b8e0737cde51cc2211201933d2  Happy to continue the discussion or rollback if you need – the commit was easier than writing out suggested changes to the various files.